core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -540,6 +1037,76 @@ class EntityApiServiceSpec extends ApiServiceSpec {
       }
   }
 
+  it should "return 200 on get deleted entity accessed by its hidden name" in withTestDataApiServices { services =>
```

Comment:
That was my thought: enable click-navigation of deleted Entities via their Submissions